exchange/src/process_spawning/spawning.cpp

```diff
@@ -46,27 +46,37 @@ get_all_users()
     return res.get<glz::json_t::object_t>();
 }
 
+std::string
+quote_id(const std::string& id)
+{
+    std::string quote_id = std::string(id);
+    std::replace(quote_id.begin(), quote_id.end(), '-', ' ');
+    return quote_id;
+}
```

Comment:
Just take a copy and the user can `std::move` if possible

```suggestion
quote_id(std::string id)
{
    std::replace(id.begin(), id.end(), '-', ' ');
    return id;
}
```